src/main/java/frc/robot/subsystems/GripperSystem.java

```diff
@@ -25,13 +25,13 @@
 public class GripperSystem extends SubsystemBase {
```

Comment:
sushil and jacobs code should be merged first, gripper system will have merge conflicts unless everything here is reverted